libcaf_core/biicode.conf

```diff
@@ -0,0 +1,42 @@
+# Biicode configuration file
+
+[requirements]
+    # Blocks and versions this block depends on e.g.
+    # user/depblock1: 3
+    # user2/depblock2(track) @tag
+
+[parent]
+	outro56/libcaf_core: 1
+[paths]
+    # Local directories to look for headers (within block)
+    /
+
+[dependencies]
+    # Manual adjust file implicit dependencies, add (+), remove (-), or overwrite (=)
+    *.hpp + *.cpp 
+    * + LICENSE LICENSE_ALTERNATIVE
```

Comment:
If you dont want to have biicode publish out the projects license then can remove this line. Also can remove the license files from the PR